packages/headless/src/api/search/search-api-client.ts

```diff
@@ -402,7 +369,7 @@ function isSearchAPIException(
   );
 }
 
-function pickNonBaseParams<Params extends BaseParam>(req: Params) {
+export function pickNonBaseParams<Params extends BaseParam>(req: Params) {
```

Comment:
It would be safer to place `pickNonBaseParams` and `unwrapError` in their own file, because atm we have a circular dependency, where:

- `search-api-client.ts` is importing from `html-api-client.ts`
- `html-api-client.ts` is importing from `search-api-client.ts`

This can lead to errors when bundling the files together,

It's nice to avoid `utils` in file names, but in this case, I can only think of `api-client-utils` for the file name 😄